tooling/templatize/config/config.go

```diff
@@ -50,19 +54,43 @@ func (cp *configProviderImpl) GetVariables(cloud, deployEnv string) (Variables,
 	return variables, err
 }
 
-func (cp *configProviderImpl) loadConfig() (*VariableOverrides, error) {
-	tmpl, err := template.ParseFiles(cp.config)
+func (cp *configProviderImpl) loadConfig(cloud, deployEnv string) (*VariableOverrides, error) {
+	vars := map[string]interface{}{
+		"ctx": map[string]interface{}{
+			"region":      cp.region,
+			"cloud":       cloud,
+			"deployEnv":   deployEnv,
+			"regionStamp": cp.regionStamp,
+			"cxStamp":     cp.cxStamp,
+		},
+	}
 
-	if err == nil {
-		var tmplBytes bytes.Buffer
+	functions := template.FuncMap{
+		"azureEventGridName": naming.AzureEventGridName,
+		"azurePostgresName":  naming.AzurePostgresName,
+		"azureKeyVaultName":  naming.AzureKeyVaultName,
+	}
```

Comment:
FuncMap should be in main.go instead of config.go. Since these functions are only used in local development, not in pipeline

Reply:
I'm confused, these methods need to be defined here, since this is where the config is parsed. So why would you want to configure them in main.go? :smile:

Reply:
can you elaborate why these would be only used in local dev?

Reply:
i'm going forward with this PR for now to unblock some work (the stamp part is the major change here) but let's discuss and we can move this as you suggested once we clarified